build/NullableEnable.props

```diff
@@ -0,0 +1,11 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- 
+    The Nullable annotations on netstandard2.0 are incomplete and incorrect in places. Ignore
+    nullable warnings on netstandard2.0 and make them errors on later target frameworks.
+  -->
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <WarningsAsErrors Condition="'$(TargetFramework)' != 'netstandard2.0'">$(WarningsAsErrors);nullable</WarningsAsErrors>
+    <NoWarn Condition="'$(TargetFramework)' == 'netstandard2.0'">$(NoWarn);nullable</NoWarn>
```

Comment:
I am curious why do we need NoWarn here? I think WarningsAsErrors wouldn't simply work for projects with netstandard2.0 tfm. Also what $(WarningsAsErrors); does here? Do we need it here?

Reply:
WarningsAsErrors makes all nullable warnings as errors. Enabled for net6.
NoWarn makes all nullable warning as nothing. Enabled for netstandard.

Reply:
> Also what $(WarningsAsErrors); does here

This causes the setting listed here to be appended to the existing `WarningsAsErrors` value, so we don't overwrite any values already set for this property.